bundles/net.tourbook/src/net/tourbook/importdata/RawDataManager.java

```diff
@@ -2796,16 +2801,15 @@ private void reimportTour_12_RequestFileFromUser(final String inSavedImportFileP
             return;
          }
 
-         Display.getDefault().syncExec(() -> {
+         Display.getDefault().syncExec(() ->
 
-            reimportTour_14_RequestFileFromUser_InUI(
+         reimportTour_14_RequestFileFromUser_InUI(
 
-                  inSavedImportFilePathName,
-                  outReimportFilePathName,
-                  tourData,
-                  importState_Process,
-                  reImportStatus);
-         });
+               inSavedImportFilePathName,
+               outReimportFilePathName,
+               tourData,
+               importState_Process,
+               reImportStatus));
       }
    }
```

Comment:
When the block {} is removed in 1, then it is not easy visible, that the reimport... method is a sub statement. I do a lot of codings and do not know every detail afterwards, so it may be possible, that this the reason why I added the block {}

![image](https://user-images.githubusercontent.com/1283445/136501772-f5214167-c6cf-41e7-af04-2406db59a73f.png)

Reply:
Reverted